openfl/display/DrawCommandType.hx

```diff
@@ -0,0 +1,29 @@
+package openfl.display;
+
+/**
+ * @author larsiusprime
+ */
+
+enum DrawCommandType {
```

Comment:
DrawCommandType, DrawCommandBuffer, DrawCommandReader should all be internal APIs, however best that is done. OpenFL users should not use these.